views/apartments/index.erb

```diff
@@ -1,5 +1,9 @@
 <ul>
-  <li><a href='/apartments/1'>Apartment 1</a></li>
-  <li><a href='/apartments/2'>Apartment 2</a></li>
-  <li><a href='/apartments/3'>Apartment 3</a></li>
+    <% @apartments.each do | apartment | %>
+  <li>
+      <a href='/apartments/<%= apartment.id %>'>
+          <%= apartment%>
```

Comment:
If you put `<%= apartment %>`, you'll probably get something like `<Apartment: 0x007fa9a95b5f68>`. Maybe you wanted `<%= apartment.address %>`?